docs/src/pages/start.rst

```diff
@@ -16,15 +18,15 @@ which configures your database, and
 
     docker run -it --env-file snowalert-{account}.envs snowsec/snowalert ./run
 
-which runs the SnowAlert functions, using auth environment variables {``SA_ROLE`` ``SA_USER`` ``PRIVATE_KEY`` and ``PRIVATE_KEY_PASSWORD``} to authenticate and operational envars {``SNOWFLAKE_ACCOUNT`` ``REGION`` ``SA_DATABASE`` ``SA_WAREHOUSE``} to create the alerts.
+which runs the SnowAlert functions, using auth environment variables {``SA_ROLE`` ``SA_USER`` ``PRIVATE_KEY`` and ``PRIVATE_KEY_PASSWORD``} to authenticate and operational envars {``SNOWFLAKE_ACCOUNT`` ``SA_REGION`` ``SA_DATABASE`` ``SA_WAREHOUSE``} to create the alerts.
 
 Finally, we have a SnowAlert Management WebUI, which you can use to edit your rules —
 
 .. code::
 
     docker run -it -p 8000:8000 --env-file snowalert-{account}.envs snowsec/snowalert-webui
 
-This container requires the runner's operational environment variables. It can also support server-side authentication with the runner's auth vars, but we suggest you leave those blank and authenticate via OAuth. This is a work in progress so please don't hesitate to ask your account manager or reach out to us with questions at snowalert@snowflake.com.
+This container may use the same environment variables as the runners to authenticate although we only recommend you do this when developing or running SnowAlert locally. When deploying to prod, please follow the instructions in :ref:`authentication`.
```

Comment:
Let's make this warning more explicit. "Snowflake OAuth is the recommended authentication method. Storing credentials in Web UI environment variables will enable anyone with web access to manage the rules and data connectors. Please don't use these environment variables in production."